marshal.go

```diff
@@ -25,8 +25,9 @@ type marshalState struct {
 	seenStructs          map[reflect.Type]bool
 }
 
-// Create a shallow clone with schema overridden.
-func (m *marshalState) withSchema(schema bool) *marshalState {
+// Create a shallow clone with schema overridden. 
+// This will cause the contents of help struct tag to be exported as comments.
+func (m *marshalState) WithSchema(schema bool) *marshalState {
```

Comment:
This can't be public as the type isn't - rather, you should create functional option (see InferHCLTags below for an example).

Reply:
This will also need some tests, and the functional option should be called `WithSchemaComments()` I think.

Reply:
Yes, you are right, I will add WithSchemaComments and some tests in stead of this.

Reply:
Thank you!

Reply:
I added the function and a test for it. i had to add an extra boolean flag. It's not ideal but it seems to work without interfering with the other flags or schemas.